lib/Launcher.js

```diff
@@ -127,7 +127,7 @@ class Launcher {
       } else {
         // Terminate chrome gracefully.
         if (process.platform === 'win32')
-          chromeProcess.kill();
+          childProcess.execSync(`taskkill /pid ${chromeProcess.pid} >nul`);
```

Comment:
I think you want `/T /F`  on the end of this. 

> /f   : Specifies that process(es) be forcefully terminated. This parameter is ignored for remote processes; all remote processes are forcefully terminated.

> /t   : Specifies to terminate all child processes along with the parent process, commonly known as a tree kill.

That's what we have in chrome-launcher, at least.

Reply:
@vsemozhetbyt  does this address the dialog? now reading the issue thread it didn't seem like the `/t /f` combo was attempted.

Reply:
I have tried `taskkill` without these key and that was OK. I do not know if `/f` is needed since [`WM_CLOSE` do the thing](https://superuser.com/questions/727724/close-programs-from-the-command-line-windows/727757#727757), I do not know either is `/t` is needed since we use `>nul` here because:

> Chrome generates many processes. The errors are caused when an process does not acknowledge the WM_CLOSE message that TASKKILL sends. Only processes with a message loop will be able to receive the message, therefore, the processes that do not have a message loop will generate that error. Most likely, these processes are the chrome extensions and plugins.

If I get this right, these keys are used in case of `WM_CLOSE` do not succeed, so maybe they will not do any harm?

Reply:
I has tried this:
```js
childProcess.execSync(`taskkill /pid ${chromeProcess.pid} /T /F >nul`);
```
and this DO NOT PREVENT the dialog. So it seems the `/F` has a privilege and force-closes the browser not-gracefully (without `WM_CLOSE` or without waiting for all the final clean-up).

Reply:
As for `/t` it seems we already send `WM_CLOSE` to the parent here, so there is no need for this key?